src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -716,6 +716,11 @@ public void onWorldTick(@NotNull TickEvent.WorldTickEvent event)
                         });
             }
 
+            if((event.world.getWorldTime() % CITIZEN_CLEANUP_TICK_INCREMENT) == 0 && !areAllColonyChunksLoaded(event))
+            {
+                Log.getLogger().info("Couldn't clean up all citizens because colonyChunks weren't loaded");
+            }
+
             //  Cleanup Buildings whose Blocks have gone AWOL
             cleanUpBuildings(event);
```

Comment:
Yeah I don't kow who created this areAllColonyChunksLoaded but that method is causing us a lot of problems on the server. We should substitute this with something else to be able to respawn lost citizens.